eel/src/event_handler.rs

```diff
@@ -118,36 +100,24 @@ impl EventHandler for LipaEventHandler {
                 amount_msat,
                 purpose,
             } => {
-                info!(
-                    "EVENT: PaymentClaimed - hash: {} - amount msat: {}",
-                    payment_hash.0.to_hex(),
-                    amount_msat,
-                );
+                let payment_hash = payment_hash.0.to_hex();
+                info!("EVENT: PaymentClaimed - hash: {payment_hash} - amount msat: {amount_msat}");
                 match purpose {
                     PaymentPurpose::InvoicePayment { .. } => {
-                        info!(
-                            "Registered incoming invoice payment for {} msat with hash {}",
-                            amount_msat,
-                            payment_hash.0.to_hex()
-                        );
+                        info!("Registered incoming invoice payment for {amount_msat} msat with hash {payment_hash}");
```

Comment:
Just for the printed message to follow the same pattern as the log regarding claiming a spontaneous payment.
```suggestion
                        info!("Claimed incoming invoice payment for {amount_msat} msat with hash {payment_hash}");
```